### ISSUE TITLE
FAQ [feature-request]

### ISSUE THREAD

**Issue by lashus** [egeloen/ivory-google-map#207]
Hello,

First of all - great job with the bundle! I'm testing it currently and so far so good :) Good job!

After browsing this bundle along with its documentation I've missed one thing - a solid "FAQ" section. It would be nice to extract some questions from Issues that were previously closed and put it onto one section.

For example the things that might be contained in there:
* How to add draggable marker
* Using autocomplete for centering map
etc.

What do you think?
@egeloen

**Comment by egeloen** [comment]
:+1: If you have some time to boostrap it, it would be awesome!

**Comment by lashus** [comment]
This evening or tommorow evening I'll release pull-request with it :)

**Comment by lashus** [comment]
#208 
ping @egeloen

**Comment by andela-odaniel** [comment]
I can't tell from the FAQ How to render a Map. It would be nice to have the steps to create and render a Map in the wiki.

**Comment by egeloen** [comment]
@andela-odaniel For rendering (everything), all is mostly self-defined here: https://github.com/egeloen/ivory-google-map/blob/master/doc/helper/index.md Isn't it enought?

**Comment by andela-odaniel** [comment]
True. It's easy to find if you know what you're looking for. Would it be possible to have a small howto on the main README? Very helpful for first timers.

**Comment by egeloen** [comment]
I have still not merged this PR and I would like to discuss it since the documentation is IMO a very important part of each project and I don't want to add extra files which does not bring real values for end users.

IMO, adding an FAQ is not a good idea if we don't organize it from the beginning as it will end with a big file plain of explanations which does not help a lot. From my side, I would prefer to add cookbook entries for real advanced use cases. Configuring a marker as dragable or attaching an event on the map (already documented in a dedicated section) is IMO not such use cases.

My POV is not the best since I know the library better than anyone else but I think if for example the event documentation can be improved instead of adding a real examples in the FAQ.

What we can add in the FAQ / Cookbook is for example how to inject an autocomplete in a map. This one is an advanced use cases the library can handle pretty easily.

Btw, @andela-odaniel, I have added a section for new comers in the README. DYT it's enought?

**Comment by lashus** [comment]
@egeloen sure cookbook should be sufficient for this case, I can understand your POV. Although there are some basics lacking in current documentation so perhaps it would be wise to add it somewhere else?

**Comment by egeloen** [comment]
I agree that the documentation is purely technicle and don't give real use cases (very true for events). You can't imagine the time it takes to me to write it already... :) If you can point me parts of the doc which are not good enough, it would help me to enhance them.

I will start by enhancing the event sections and for cookbook entries, I will start with the autocomplete injected into a map.

**Comment by lashus** [comment]
As soon as i  get back to the project that utilizes your library i will add sth from myself, currently i will have trouble to find time :( perhaps someone else could be more helpful?

Thanks for all :)
